src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+@import 'variables';
+
+.product {
```

Comment:
If the file has the name 'card' so it should be styles for the card in it. So rename product classes or rename the file to match the name